homeassistant/components/habitica/coordinator.py

```diff
@@ -0,0 +1,55 @@
+"""DataUpdateCoordinator for the Habitica integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import timedelta
+import logging
+
+from aiohttp import ClientResponseError
+from habitipy.aio import HabitipyAsync
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class HabiticaData:
+    """Coordinator data class."""
+
+    user: dict
```

Comment:
A dict of what?

Reply:
could be anything, but the keys are strings ^^